modules/_template.nix

```diff
@@ -1,8 +1,4 @@
 { self, lib, ... }: {
-  perSystem = { config, self', inputs', pkgs, ... }: {
-
-  };
-  flake = {
-
-  };
+  perSystem = { config, self', inputs', pkgs, ... }: { };
+  flake = { };
```

Comment:
formatting the template like this makes it more confusing I believe.

Reply:
just a result of formatting all nix files in one generic command

Reply:
ideally we change the cachix action to tolerate running with an untrusted user. the issue i've seen is that the file that captures the current store paths isn't created, which leads to it not being found when attempting to diff and push.